FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_IPv6_Demo/IPv6_Multi_WinSim_demo/ReadMe.md

```diff
@@ -46,21 +46,50 @@ the location of source files:
 You can changes these directory to let the proyejct work with a different
 source tree.
 
-The Multiple Interface Visual studio demo showcases the Multiple Interfaces (or
-rather the multiple endpoints) functionality of the FreeRTOS+TCP
-IPv6/multi-interface library. The Windows Simulator environment doesn't actually
-have multiple interfaces which can be used by FreeRTOS and thus, this demo shows
+## Introduction
+
+The IPv6_Multi_WinSim Visual studio demo showcases the Multiple Interfaces (or
+rather the multiple endpoints) functionality of the FreeRTOS+TCP library.
+The Windows Simulator environment doesn't actually have multiple
+interfaces which can be used by FreeRTOS and thus, this demo shows
 the use of different endpoints which will be resolved by the OS having multiple
 interfaces. It shows that the library will use different endpoints (IP-addresses)
 to connect to IP-addresses on different subnets (or using different netmasks).
-The instructions for the full Windows demo are still relevant though as they
-describe how to set up the WinPCap development environment, how to set the IP
-address, and other such items. Note that, as delivered, configUSE_DHCP is set to 0,
-so a static IP address is used. The instructions are provided on the following URL,
-see the "Hardware Setup" section:
+
+Note that, as delivered, configUSE_DHCPv6 is set to 0, so a static IP address is used. 
+
+## Prerequisites
+
+The demo is inline with the [FreeRTOS_Plus_TCP_Minimal_Windows_Simulator](https://github.com/FreeRTOS/FreeRTOS/tree/main/FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_Minimal_Windows_Simulator)
+
+The initial setup instructions are provided on the following URL, see the 
+"Hardware Setup" and "Software Setup" :
 http://www.FreeRTOS.org/FreeRTOS-Plus/FreeRTOS_Plus_TCP/examples_FreeRTOS_simulator.html
 
-The IPv6_Multi_WinSim_demo demo performs some basic network activities:
+Additionally in FreeRTOSConfig.h, we also need to set configECHO_SERVER_ADDR_STRING as an IPv6 Server
+IP address for IPv6 validation, it can be updated to IPv4 address for IPv4 validaitons.
```

Comment:
IPv4 validaitons --> IPv4 validation

Reply:
I have added for both here. Is there any issue?

Reply:
I was mentioning the spelling for validations

Reply:
Sure